### PR TITLE
feat(core): allow array of ids as updateNodeInternals arg

### PR DESCRIPTION
**Pull request by bcakmakoglu**
# What's changed?

- In addition to a single id, also allow for an array of ids to be passed to `updateNodeInternals`

### CODE REVIEW

File: packages/core/src/hooks/useUpdateNodeInternals.ts [hunk 1]
```diff
@@ -6,12 +6,18 @@ import type { UpdateNodeInternals } from '../types';
...
     const { domNode, updateNodeDimensions } = store.getState();
     const nodeElement = domNode?.querySelector(`.react-flow__node[data-id="${id}"]`) as HTMLDivElement;
 
     if (nodeElement) {
-      requestAnimationFrame(() => updateNodeDimensions([{ id, nodeElement, forceUpdate: true }]));
+      const updateIds = Array.isArray(id) ? id : [id];
+
+      requestAnimationFrame(() => {
+        updateIds.forEach((id) => {
+          updateNodeDimensions([{ id, nodeElement, forceUpdate: true }]);
```
Comment (moklick) [inline packages/core/src/hooks/useUpdateNodeInternals.ts:18]: the nodeElement would always be the same node. You also need to call `domNode?.querySelector(`.react-flow__node[data-id="${id}"]`)` multiple times. It's also possible to pass an array with ids and nodeElements to `updateNodeDimensions`.

**Review by moklick — CHANGES_REQUESTED** [review]

File: packages/core/src/hooks/useUpdateNodeInternals.ts (hunk 1 above)
Comment (bcakmakoglu) [reply packages/core/src/hooks/useUpdateNodeInternals.ts:18]: Ah yeah, that makes sense - I'll change that
Thanks for keeping an eye out, I didn't think too much doing this PR and it shows 🤣

Comment (moklick) [reply packages/core/src/hooks/useUpdateNodeInternals.ts:18]: querySelector should go in the `forEach` so that `nodeElement` matches the actual node.

**Review by moklick — CHANGES_REQUESTED** [review]

File: packages/core/src/hooks/useUpdateNodeInternals.ts (hunk 1 above)
Comment (bcakmakoglu) [reply packages/core/src/hooks/useUpdateNodeInternals.ts]: Done 👍

**Comment by moklick** [comment]
Thanks @bcakmakoglu 🙌

**Pull request merged**